var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -80,28 +81,21 @@ def check(self):
 
     for ver in ['3.5.0', '3.7.0', '3.8.0', '3.9.0', '3.10.0', '4.0.0', '4.1.0',
                 '4.2.0', '4.3.0', '4.3.1', '4.5.0', '4.5.2', '5.0.0', '5.0.2',
-                '5.1.0', '5.1.3']:
+                '5.1.0', '5.1.3', '5.2.0']:
         depends_on('hip@' + ver,                         when='@' + ver)
         depends_on('llvm-amdgpu@' + ver,  type='build',  when='@' + ver)
         depends_on('rocminfo@' + ver,     type='build',  when='@' + ver)
 
-    for ver in ['3.5.0', '3.7.0', '3.8.0', '3.9.0']:
-        depends_on('rocm-smi@' + ver, type='build', when='@' + ver)
-
-    for ver in ['4.0.0', '4.1.0', '4.2.0', '4.3.0', '4.3.1', '4.5.0', '4.5.2',
-                '5.0.0', '5.0.2', '5.1.0', '5.1.3']:
-        depends_on('rocm-smi-lib@' + ver, type='build', when='@' + ver)
-
-    # This is the default library format since 3.7.0
-    depends_on('msgpack-c@3:', when='@3.7:')
-
     depends_on('python@3.6:', type='build')
-    depends_on('py-virtualenv', type='build')
-    depends_on('perl-file-which', type='build')
-    depends_on('py-pyyaml', type='build')
-    depends_on('py-wheel', type='build')
-    depends_on('py-msgpack', type='build')
-    depends_on('py-pip', type='build')
+    with when('+tensile'):
```

Comment:
please edit this changes as these were part of #31414

Reply:
That PR has now been merged.

Reply:
Reverted the changes in rocblas which merged as part of PR31414